pyvista/plotting/themes.py

```diff
@@ -1754,6 +1756,23 @@ def opacity(self, opacity: float):  # numpydoc ignore=GL08
         _check_range(opacity, (0, 1), 'opacity')
         self._opacity = float(opacity)
 
+    @property
+    def edge_opacity(self) -> float:  # numpydoc ignore=RT01
+        """Return or set the edges opacity.
```

Comment:
Should we also put in the version compatibility note here too?

Reply:
```suggestion
        """Return or set the edges opacity.

        .. note::
            `edge_opacity` uses ``SetEdgeOpacity`` as the underlying method which
            requires VTK version 9.3 or higher. If ``SetEdgeOpacity`` is not
            available, `edge_opacity` is set to 1.
```